staubli_rx160_gazebo/package.xml

```diff
@@ -0,0 +1,40 @@
+<package>
+  <name>staubli_rx160_gazebo</name>
+  <version>0.0.1</version>
+  <description>
+    <p>
+      ROS-Industrial Gazebo support package for the Staubli RX160 (and variants).
+    </p>
+    <p>
+      This package contains the configuration data and launch files required
+      to simulate the Staubli RX160 manipulator in Gazebo. This includes the base
+      model and the RX160L.
+    </p>
+    <p>
+      Before using any of the configuration files included in this package, be
+      sure to check they are correct for the particular robot model and
+      configuration you intend to use them with.
+    </p>
+  </description>
+  <author email="murilo.martins@ocado.com">Murilo Martins (Ocado Technology)</author>
+  <maintainer email="murilo.martins@ocado.com">Murilo Martins (Ocado Technology)</maintainer>
```

Comment:
Please add at least one maintainer from the core ROS-Industrial team. We try to avoid having too many different contact points for support. You can add my contact info there (see pkgs in the `staubli` repository).

Also: we normally don't repeat the `email` attribute whenever it is the same for both `author` and `maintainer` elements.